src/SFA.DAS.ApprenticeFeedback.Api/Startup.cs

```diff
@@ -87,12 +87,19 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 app.UseDeveloperExceptionPage();
             }
+
+            if (env.IsProduction())
```

Comment:
Technically this is correct as IsProduction is true for any of the Azure environments AT, TEST, TEST2, PP, PRD, MO, DEMO - but it reads like it would be for PRD only - you won't find many example of other code doing that - best to use the "if(IsDevelopment) else" style like you have for the others